src/software/sensor_fusion/ssl_protobuf_reader.cpp

```diff
@@ -1,35 +1,11 @@
-#include "software/backend/input/network/ssl_protobuf_reader.h"
+#include "software/sensor_fusion/ssl_protobuf_reader.h"
```

Comment:
this entire file is really nicely cleaned up, nice work :fire:

Reply:
🚀